component/admin/models/webservice.php

```diff
@@ -73,6 +73,29 @@ class WebservicesModelWebservice extends JModelAdmin
 	 */
 	public function __construct($config = array())
 	{
+		if (empty($config['filter_fields']))
+		{
+			$config['filter_fields'] = array(
+				'published', 'w.published',
+				'client', 'w.client',
+				'state', 'w.state',
+				'path', 'w.path',
+			);
+		}
+
+		try
+		{
+			$container = (new Joomla\DI\Container)
+				->registerServiceProvider(new Joomla\Webservices\Service\ConfigurationProvider)
+				->registerServiceProvider(new Joomla\Webservices\Service\DatabaseProvider);
+		}
+		catch (\Exception $e)
+		{
+			throw new RuntimeException('Help!', 500);
```

Comment:
When you improve these exception messages, also attach the caught exception to the new exception you're throwing:

```php
throw new RuntimeException('George will improve this message', 500, $e);
```